tests/baselines/reference/genericCallsWithoutParens.js

```diff
@@ -17,4 +17,4 @@ var C = /** @class */ (function () {
     }
     return C;
 }());
-var c = new C(); // parse error
+var c = new C; // parse error

```

Comment:
Looks like there's a difference here because we no longer create a missing argument list. I *think* that's probably fine.